crates/ruff_python_formatter/src/core/helpers.rs

```diff
@@ -32,6 +37,83 @@ pub fn is_radix_literal(content: &str) -> bool {
         || content.starts_with("0X")
 }
 
+/// Expand the range of a compound statement.
+///
+/// `location` is the start of the compound statement (e.g., the `if` in `if x:`).
+/// `end_location` is the end of the last statement in the body.
+pub fn expand_indented_block(
```

Comment:
Ideally, in the future, this could be handled by a parser, rather than with this separate conversion pass.